Packs/GroupIB_ThreatIntelligenceAttribution/IncidentFields/incidentfield-GIB_Link_List.json

```diff
@@ -1,29 +0,0 @@
-{
```

Comment:
Why do we need to delete them? Are they no longer retrieved by the integration or something else? Is there a replacement?

Reply:
This field will no longer be used, it has been replaced by incidentfield-GIB_Link_List_Table.json